.dockerignore

```diff
@@ -11,10 +11,14 @@ web-api/.dynamodb
 web-api/.nyc_output
 web-api/.serverless
 web-api/coverage
-web-api/.cache
 web-api/dist
+web-api/.cache
```

Comment:
I have been unable to batch download cases, as covered by [story 1282](https://github.com/flexion/ef-cms/issues/1282). This is true across browsers, in any session that has cases. This is while running the site locally, using `docker-run.sh`. (@rachaelparris has been able to replicate this, luckily.) Please address this problem.

Reply:
Updated.

Reply:
Works great — thank you!